pmdarima/arima/arima.py

```diff
@@ -1196,8 +1197,12 @@ def plot_diagnostics(self, variable=0, lags=10, fig=None, figsize=None):
         ax = fig.add_subplot(222)
         # temporarily disable Deprecation warning, normed -> density
         # hist needs to use `density` in future when minimum matplotlib has it
+        # normed keyword arugment is no longer supported in matplotlib since version 3.2.0
```

Comment:
In addition to the discussion that is happening on the actual implementation, our linter is complaining that this line is too long (>79 characters), hence why the tests are failing

Reply:
I will edit the line to be less <=79 characters